app/containers/TopBarContainer.js

```diff
@@ -22,6 +22,9 @@ export default class TopBarContainer extends Component<Props> {
       wallet={stores.substores[environment.API].wallets.active}
       currentRoute={app.currentRoute}
       formattedWalletAmount={formattedWalletAmount}
+      theme={{
```

Comment:
given the word `theme` is already used for a different concept, can we rename this?

Reply:
Renamed into `themeProperties`. The logic was that once we migrate to proper "themes" - they should be passed as objects with this identicon saturation level optionally defined.